src/lib/shiori-loader.ts

```diff
@@ -0,0 +1,62 @@
+/**
+ * 'fs' - node.js/BrowserFS のファイルシステム
+ */
+export type FileSystemLike = any;
+
+/**
+ * SHIORIサブシステム
+ */
+export interface Shiori {
+  /**
+   * SHIORI/2.x, SHIORI/3.x load()
+   * @param dirpath 'ghost/master'へのフルパス 末尾は必ずパスセパレータ(環境依存で'/'または'\')
+   * @return ステータスコード
+   */
+  load(dirpath: string): Promise<number>;
+  /**
+   * SHIORI/2.x, SHIORI/3.x unload()
+   * @return ステータスコード
+   */
+  unload(): Promise<number>;
+  /**
+   * SHIORI/2.x, SHIORI/3.x request()
+   * @param request SHIORI/2.x, SHIORI/3.x リクエスト
+   * @return SHIORI/2.x, SHIORI/3.x レスポンス
+   */
+  request(request: string): Promise<string>;
+}
+
+/** SHIORI判定関数 */
+export interface ShioriDetector {
```

Comment:
__[tslint]__
* interface name must be a capitalized I

<!-- Comment by [SideCI](https://sideci.com) -->